### PR TITLE
[UID-154] Look up capabilities/capabilitySets by permission displayName in Capabilities utility

### PR DESCRIPTION
**Pull request by ryandberger**
- Fulfills [UID-154](https://folio-org.atlassian.net/browse/UID-154).
- Relies on [STCOR-867](https://folio-org.atlassian.net/browse/STCOR-867) which has been merged.
- Tool adds drop-down to toggle between searching by `permissionName` or `permissionDisplayName`
- Also added support for searching by pressing the 'Enter'/'Return' key.

https://github.com/user-attachments/assets/6039e8bb-6842-4bbb-9868-f505fd7877f7

### CODE REVIEW

**Review by zburke — APPROVED** [review]
Code changes look good but lint is cranky, so please fix that before merging. 

You can avoid specifically calling `handleKeyDown` [by using a `<Form>` wrapper with a `handleSubmit` render-prop](https://codesandbox.io/p/sandbox/github/final-form/react-final-form/tree/master/examples/simple?file=%2Findex.js). The current approach works but feels a little off since since submit-on-enter is a default behavior for forms, if not for individual fields. I guess a function attached to button isn't that different than a function attached to a form, but the latter feels a little more standard and would extend this functionality over all text-fields if we add more. Ultimately your call.

**Comment by ryandberger** [comment]
> Code changes look good but lint is cranky, so please fix that before merging.
> 
> You can avoid specifically calling `handleKeyDown` [by using a `<Form>` wrapper with a `handleSubmit` render-prop](https://codesandbox.io/p/sandbox/github/final-form/react-final-form/tree/master/examples/simple?file=%2Findex.js). The current approach works but feels a little off since since submit-on-enter is a default behavior for forms, if not for individual fields. I guess a function attached to button isn't that different than a function attached to a form, but the latter feels a little more standard and would extend this functionality over all text-fields if we add more. Ultimately your call.

I tried wrapping with `<Form>` before and the page would refresh when pressing Enter. I can give it another go since this is an established pattern and maybe I missed something the first time around. If I get the same behavior, I'll plan to stick with `handleKeyDown` in the interest of time.

**Comment by ryandberger** [comment]
> > Code changes look good but lint is cranky, so please fix that before merging.
> > You can avoid specifically calling `handleKeyDown` [by using a `<Form>` wrapper with a `handleSubmit` render-prop](https://codesandbox.io/p/sandbox/github/final-form/react-final-form/tree/master/examples/simple?file=%2Findex.js). The current approach works but feels a little off since since submit-on-enter is a default behavior for forms, if not for individual fields. I guess a function attached to button isn't that different than a function attached to a form, but the latter feels a little more standard and would extend this functionality over all text-fields if we add more. Ultimately your call.
> 
> I tried wrapping with `<Form>` before and the page would refresh when pressing Enter. I can give it another go since this is an established pattern and maybe I missed something the first time around. If I get the same behavior, I'll plan to stick with `handleKeyDown` in the interest of time.

Try as I might, with the new component style I can't simply reuse logic found in other tools such as `Passwd.js`, but even following the prescription in https://react.dev/reference/react-dom/components/form I still find the page refreshes when pressing Enter, so I am going to stick with the current pattern.

**Pull request merged**
